src/aap_eda/services/auth.py

```diff
@@ -16,7 +16,7 @@
 
 from rest_framework_simplejwt.tokens import RefreshToken
 
-from aap_eda.core.models import User
+from aap_eda.core.models.user import User
```

Comment:
It was changed to avoid a circular import issue involving `models/__init__.py` (`Activation -> ContainerableMixin -> create_jwt_token -> User`.
It's not ideal but, short-term, it solves the issue without having to perform a more involved location restructuring.